_notebooks/2020-03-27-covid19-progress-projections.ipynb

```diff
@@ -0,0 +1,2122 @@
+{
```

Comment:
<p>Are these toy examples or is this the Actual model used for this Dashboard?  The wording / language could be clarified here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/github/covid19-dashboard/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>These are the actual numbers. I've added clarifications.</p><p> </p>